sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -109,6 +109,16 @@ export interface ExponentialRetryPolicyOptions {
     retryDelayInMs?: number;
 }
 
+// @public
+export type FailoverHostDelegate = (retryState: RetryInformation, options: DefaultRetryPolicyOptions) => Iterator<FailoverHostState | undefined, void, RetryInformation>;
```

Comment:
maybe `options` here should be `InternalPipelineRetryOptions` just in case `DefaultRetryPolicyOptions` adds some options specific to that retry policy rather than common to all retry policies?

Reply:
This makes sense. I'll go ahead and update that.